src/Themes/Fonts.ts

```diff
@@ -1,173 +1,49 @@
-import { normalize } from "./FontHelper";
+import { normalize } from "../Helpers/sizeHelper";
+import { getColors, ColorThemeNames } from "./Colors";
 
-const type = {
-  base: "System",
-  bold: "Avenir-Black",
-  emphasis: "HelveticaNeue-Italic",
+export const getFonts = (theme: ColorThemeNames) => {
+  const color = getColors(theme).defaultText;
+  return {
+    LargeTitle: {
+      fontFamily: "Roboto-Medium",
```

Comment:
How it will work on iOS, which does not have Roboto font preinstalled?

And on Web?

Reply:
will use default font

Reply:
fixed

Reply:
So we've both decided to use the system's default font not to think about assets sharing between dependency and the consumer.

Need to remove those fontName styles.